### DISCUSSION TITLE
How Github works with communicating to project admins/OP

### DISCUSSION THREAD

**Discussion by arvobowen** [community/community#23035] [New to GitHub]
<p>I’m new to Github (though I have had an account for some time).  I’m just now starting to get interested in storing my source on Github and trying to learn about the community aspect etc.</p>
<p><strong>More Information Than You Need (Backstory)</strong></p>
<p>I have this one project I’m currently looking at <a href="https://github.com/Seeed-Studio/IRSendRev" target="_blank" rel="nofollow noopener">https://github.com/Seeed-Studio/IRSendRev</a> because I’m trying to create something that will capture/store an IR signal (like from a TV remote control) and then allow me to later send the command back out to a device.  The reason I need to have something like this is that I have a programmable/learning remote that works great with all other remotes except one TV remote that seems to send out short IR bursts.  The learning remote will not pick up the signal because it does not send long enough.  So I want to build something that will take in the short burst and allow me to send it out in a longer burst.</p>
<p>The point of this post/question is to figure out more about GitHub.  I would like to know how I could contact the original poster of the project or simply ask a question in regards to the project to see if I can get a little more information on how it is supposed to work, etc.</p>
<p>Is this possible?  Or is there a better place on GitHub to achieve what I’m trying to do?</p>
<p>Thanks!!</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/arvobowen">@arvobowen</a>, welcome to Github!</p>
<p>Most commonly when asking questions about a project you can use the <a href="https://help.github.com/en/articles/creating-an-issue" rel="nofollow noopener">issues section</a>. It’s mainly used to track bugs, feature requests, etc…, but it can also be used to ask questions. Of course, it’s always good to check beforehand to see if the project you want to ask a question about has another means for support, such as a forum or e-mail you can send your queries to, which could typically be found in the readme file or wiki. If not, then opening an issue would be the best way to get in contact.</p>

**Comment by arvobowen** [comment]
<p>Perfect!  That’s what I was thinking but I didn’t want to do something that was in bad taste on GitHub when trying to fit in and all.</p>
<p>Thanks for the really fast response!</p>

**Comment by lee-dohm** [comment]
<p>In addition to <a class="mention" href="/sethclydesdale">@sethclydesdale</a>’s answer, you should look for <a href="https://help.github.com/en/articles/adding-support-resources-to-your-project" rel="nofollow noopener">a document titled <code>SUPPORT</code> in the repository</a>. It’s an optional repository metadata document that tells people where to find support for the repository’s contents. If that file isn’t present, you should check the <code>README</code>, and then if there’s no indicator for where to find support, filing an issue is the best approach.</p>

**Comment by Munyola** [comment]
<p>Welcome brother,thanks for your question. Fast of all Github is home of education, look well if you are ready to contribute and tackles on issues of projects start with your favorite softwares from their you will found automatic way to make contribute on any of project through pull request. Otherwise if you fall into any project problem contact us on github.community.<br>
Thanks.</p>
